code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -304,18 +295,20 @@
 								add_attack_logs(src, C, "[src] has grabbed [I] out of [C]'s hand with a tentacle")
 								on_hit(I) //grab the item as if you had hit it directly with the tentacle
 								return 1
-							else
-								to_chat(firer, "<span class='danger'>You can't seem to pry [I] out of [C]'s hands!</span>")
-								add_attack_logs(src, C, "[src] tried to grab [I] out of [C]'s hand with a tentacle, but failed")
-								return 0
-						else
-							to_chat(firer, "<span class='danger'>[C] has nothing in hand to disarm!</span>")
-							return 0
+							to_chat(firer, "<span class='danger'>You can't seem to pry [I] out of [C]'s hands!</span>")
+							add_attack_logs(src, C, "[src] tried to grab [I] out of [C]'s hand with a tentacle, but failed")
```

Comment:
Again here, `src` is gonna cause issues with it being the projectile. Please go over all the src instances in this file to make sure you are using them in the right places. (Ther are fine to use in some instances)

Reply:
SRC fixed